CHANGELOG.md

```diff
@@ -2,6 +2,58 @@
 
 Notable changes to QMCPACK are documented in this file.
 
+## [Unreleased]
+
+
+This release includes GPU support for the AFQMC implementation,
+Quantum Espresso v6.4 support, and in the real-space code makes the
+structure-of-arrays (SoA) code path the default. A large number of
+feature refinements, bugfixes, testing improvements and source code
+cleanup have been performed.
+
+### QMCPACK
+
+* The improved structures of arrays (SoA) build is now the
+  default. This is generally significantly faster and uses less memory
+  than the AoS build due to better algorithms, but does not yet have
+  the full range of functionality. The older AoS build can be selected
+  with -DENABLE_SOA=0.
+* AFQMC code fully supports GPU acceleration via NVIDIA CUDA.
```

Comment:
1. Need to mention -D ENABLE_CUDA=1 for AFQMC instead of the usual -DQMC_CUDA=1
2. Remind users only GNU works with AFQMC no matter using GPU or not. Intel doesn't work and I have no idea about Clang.